src/transcoder/transcodedialog.cpp

```diff
@@ -370,6 +375,19 @@ void TranscodeDialog::SetFilenames(const QStringList &filenames) {
     QTreeWidgetItem *item = new QTreeWidgetItem(ui_->files, QStringList() << name << path);
     item->setData(0, Qt::UserRole, filename);
   }
+}
+
+void TranscodeDialog::SetImportFilenames(const QStringList &filenames, const QString &import_dir) {
```

Comment:
This functionality could be merged with `TranscodeDialog::SetFilenames(const QStringList &filenames)` by adding a default parameter value for `import_dir` and adding some conditional logic. I opted to separate the two unless the alternative is preferred.